app/views/admin/promotional_feature_items/_promotional_feature_item.html.erb

```diff
@@ -1,26 +1,105 @@
-<%= content_tag_for(:section, promotional_feature_item, class: 'well') do %>
-  <% unless promotional_feature_item.image.s300.url.blank? %>
-    <%= image_tag promotional_feature_item.image.s300.url, alt: promotional_feature_item.image_alt_text %>
-  <% end %>
-
-  <% if promotional_feature_item.youtube_video_url.present? %>
-    <p>
-      <%= link_to promotional_feature_item.youtube_video_url, promotional_feature_item.youtube_video_url %>
-    </p>
-  <% end %>
-
-  <% if promotional_feature_item.title.present? %>
-    <p><%= link_to_unless promotional_feature_item.title_url.blank?, promotional_feature_item.title, promotional_feature_item.title_url %></p>
-  <% end %>
-  <p>
-    <%=format_with_html_line_breaks promotional_feature_item.summary %>
-  </p>
-  <p>
-    <% promotional_feature_item.links.each do |link| %>
-      &mdash; <%= link_to link.text, link.url %><br />
-    <% end %>
-  </p>
-
-  <%= link_to "Edit", edit_admin_organisation_promotional_feature_item_path(@organisation, @promotional_feature, promotional_feature_item), class: "btn btn-default"%>
-  <%= link_to "Delete", confirm_destroy_admin_organisation_promotional_feature_item_path(@organisation,@promotional_feature, promotional_feature_item), class: 'btn btn-danger btn-sm' %>
-<% end %>
+<%= render "components/summary_card_component", {
+  title: "Feature item #{promotional_feature_item.id}",
```

Comment:
Does it make sense to number these sequentially rather than using the ids? So we get `Feature item 1` `Feature item 2` etc.

<img width="527" alt="image" src="https://github.com/alphagov/whitehall/assets/42515961/9ab9fce4-b4ee-49fa-acf1-658dd89a3b8c">

rather than using the id which might seem random to the user

<img width="581" alt="image" src="https://github.com/alphagov/whitehall/assets/42515961/a3b5eeed-233c-47d9-90dc-3f3bd3350c62">




```
  title: "Feature item #{promotional_feature_item.promotional_feature.promotional_feature_items.find_index(promotional_feature_item) + 1}",
```